RELEASE.md

```diff
@@ -48,6 +49,58 @@ Check issues at https://github.com/Icinga/icinga2
 For minor versions you need to manually backports any and all commits from the
 master branch which should be part of this release.
 
+### Windows Dependencies <a id="windows-dependencies"></a>
+
+In contrast to Linux, the Windows dependencies (at least Boost and OpenSSL)
+are neither updated independently of Icinga itself by instance administrators,
+nor their latest versions available for the OS are fetched at package build time.
```

Comment:
I find this paragraph a bit hard to read. Maybe go with something like "... Windows dependencies are bundled with the Icinga installer and are not updated automatically"?